### PR TITLE
Paperwork Error Event

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Added the Bureaucratic Error event, which replaces the overflow role with a random job.
/:cl:

Randomly sets the overflow role (the infinite slot one, default is Assistant) to any job in the game.

Why: I think rare events that disrupt the round without directly killing people or blowing things up have strong potential for EMERGENT stories and gameplay.

The one time I ran this manually about 9 chaplains held elaborated rituals in the chapel, then declared alchohol sinful and wrecked the bar before being shotgunned to death by cargo.

### CODE REVIEW

File: code/modules/events/bureaucratic_error.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+/datum/round_event/bureaucratic_error
+	announceWhen = 1
+
+/datum/round_event/bureaucratic_error/announce(fake)
+	priority_announce("A recent bureaucratic error in the Organic Resources Department may result in personnel shortages in some departments and redundant staffing in others.", "Paperwork Mishap Alert")
+
+/datum/round_event/bureaucratic_error/start()
+	var/list/all_jobs = get_all_jobs()
+	picked_job = pick(all_jobs)
```
[the remark this answers is not in the document]
Comment (ninjanomnom) [reply code/modules/events/bureaucratic_error.dm]: You can

**Comment by Iamgoofball** [comment]
You should make this close other job slots too.

**Comment by evsey9** [comment]
We already have a job for that, it's call the head of personnel

**Comment by Bluespace** [comment]
This is getting out of hand, now there's two of them! (Assistant to head of security, 2018)

**Comment by kevinz000** [comment]
shoudl be a way to set overflow jobs at id consolse imo

**Comment by AnturK** [comment]
Feels it would be more interesting without the announcement ? Or maybe not.

**Comment by Jalleo** [comment]
Maybe instead a announcement on the command channel?

File: code/modules/events/bureaucratic_error.dm [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	name = "Bureaucratic Error"
+	typepath = /datum/round_event/bureaucratic_error
+	max_occurrences = 1
+	weight = 5
+
+/datum/round_event/bureaucratic_error
+	announceWhen = 1
+
+/datum/round_event/bureaucratic_error/announce(fake)
+	priority_announce("A recent bureaucratic error in the Organic Resources Department may result in personnel shortages in some departments and redundant staffing in others.", "Paperwork Mishap Alert")
```
Comment (JJRcop) [inline code/modules/events/bureaucratic_error.dm]: Append "This issue should be resolved shortly." even though the event is permanent.

**Comment by KorPhaeron** [comment]
@AnturK players knowing its an event/about to happen might help the HoP or whoever prepare rather than spamming ahelps

That and the announcement makes it feel like an inworld event with an explanation, or at least that was my thinking

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**